webapp/src/containers/MinimapContainer.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import { selectors } from '../reducers'
+
+import Minimap from '../components/Minimap.js'
+
+class MinimapContainer extends React.Component {
+  static propTypes = {
+    range: PropTypes.shape({
+      minX: PropTypes.number,
+      minY: PropTypes.number,
+      maxX: PropTypes.number,
+      maxY: PropTypes.number
+    })
+  }
+
+  constructor(...args) {
+    super(...args)
+    this.onDragEnd = newRange => ({})
```

Comment:
What's this? and why not a class method?

Reply:
Hey, sorry, this should be out. I left it there to implement click+drag to move around, but never got to it.